### PR TITLE
Sunset bintray 2.1.x

### PR DESCRIPTION
**Pull request by ignasi35**
Manual backport of #10803

### CODE REVIEW

**Comment by ignasi35** [comment]
Blocked by https://github.com/playframework/interplay/pull/161

File: project/plugins.sbt [hunk 1]
```diff
@@ -13,7 +13,7 @@ val webjarsLocatorCore = "0.43"
 val sbtHeader          = "5.2.0"
 val scalafmt           = "2.0.1"
 val sbtTwirl: String   = sys.props.getOrElse("twirl.version", "1.5.1") // sync with documentation/project/plugins.sbt
-val interplay: String  = sys.props.getOrElse("interplay.version", "2.1.6")
+val interplay: String  = sys.props.getOrElse("interplay.version", "2.1.7-SNAPSHOT")
```
Comment (marcospereira) [inline project/plugins.sbt:16]: Just a reminder to replace it later. 👀

**Review by marcospereira — APPROVED** [review]
LGTM.

I guess the same comment I've made in the other PR applies here: let's run a snapshot release and see how it goes to validate all the changes are good.

File: project/plugins.sbt (hunk 1 above)
Comment (ignasi35) [reply project/plugins.sbt:16]:
```suggestion
val interplay: String  = sys.props.getOrElse("interplay.version", "2.1.8")
```

Comment (mkurz) [reply project/plugins.sbt:16]: https://repo1.maven.org/maven2/com/typesafe/play/interplay_2.12_1.0/2.1.8/

**Comment by ignasi35** [comment]
> I guess the same comment I've made in the other PR applies here: let's run a snapshot release and see how it goes to validate all the changes are good.

I think I'd rather run a release of 2.8.8-RC1 to be extras-sure that production releases are all 100% properly set. Publishing snapshots to Sonatype requires the version to end on `-SNAPSHOT` and play versions don't fit that pattern because we use dynver so releasing a snapshots we'd be fighting the wrong wars.

**Pull request merged**